packages/tldraw/src/hooks/useKeyboardShortcuts.tsx

```diff
@@ -15,6 +15,27 @@ export function useKeyboardShortcuts(ref: React.RefObject<HTMLDivElement>) {
     [ref]
   )
 
+  React.useEffect(() => {
+    if (!app) return
+    const handlePaste = (e: ClipboardEvent) => {
+      let items = e.clipboardData?.items ?? []
```

Comment:
It works so well. The solution is simple & standard. Not sure why I am not able to make it work when I tried long time back.

Thanks a lot for this PR :)